File: dataset_reduzido/codenet/Java/Code/s897176066.java
import java.util.*;
public class Main{
  public static void main(String[] args){
    Scanner sc =new Scanner(System.in);
    int N =sc.nextInt();
    long A[] =new long[N];
    for(int i=0;i<N;i++){
      A[i]=sc.nextLong();}
    Arrays.sort(A);
    for(int j=1;j<N;j++){
      if(A[j]==A[j-1]){System.out.println("NO");System.exit(0);}}
    System.out.println("YES");
  }
}

File: output/Llama-2-100/codenet/Java/Python/s897176066.py
import sys

N = int(sys.stdin.readline())
A = [long(sys.stdin.readline()) for _ in range(N)]

A.sort()

for j in range(1, N):
    if A[j] == A[j-1]:
        print("NO")
        sys.exit(0)

print("YES")
